testfiles/CMakeLists.txt

```diff
@@ -330,6 +330,8 @@ ADD_SIMULATION_TEST(IDF_FILE PlateHeatExchanger.idf EPW_FILE USA_FL_Tampa.Intl.A
 ADD_SIMULATION_TEST(IDF_FILE Plenum.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 ADD_SIMULATION_TEST(IDF_FILE PlenumwithRetAirHeatGain.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 ADD_SIMULATION_TEST(IDF_FILE PondGroundHeatExchanger.idf EPW_FILE USA_AZ_Phoenix-Sky.Harbor.Intl.AP.722780_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE DOAToPTAC.idf EPW_FILE USA_FL_Miami.Intl.AP.722020_TMY3.epw)
```

Comment:
@Nigusse Thanks for renaming these.  The new rows need to be moved no to keep this cmakelist in alphabetical order.  But before that, we need to decide if we want to keep all of these.  It's fantastic that we have a test file for each combination of DOA mixer and zoneHVAC type, but this now gives us 8 DOATo* test files that all get packaged in ExampleFiles.  

@Myoldmopar Do we want to move some of these to _DOATo* to reduce the number of ExampleFiles?